### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
Hi, I'm making updates for Open Collective. Either you or another core contributor signed this repository up for Open Collective. This pull request adds financial contributors from your Open Collective https://opencollective.com/aiogram ❤️

  What it does:
  - adds a badge to show the latest number of financial contributors
  - adds a banner displaying contributors to the project on GitHub
  - adds a banner displaying all individuals contributing financially on Open Collective
  - adds a section displaying all organizations contributing financially on Open Collective, with their logo and a link to their website

P.S: As with any pull request, feel free to comment or suggest changes.

  Thank you for your great contribution to the Open Source community. You are awesome! 🙌
  And welcome to the Open Collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other Open Source sustainers!

### CODE REVIEW

**Comment by JrooTJunior** [comment]
Why this change is pushed into master branch? It should be in default branch and is `dev-2.x`.
And you clone from `dev-2.x` but push into `master`

Here `master` branch has the stable version from PyPi
`dev-2.x` is default branch in GitHub and all new changes should be pushed into this branch.

**Comment by JrooTJunior** [comment]
I'll change the base

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # AIOGram
 
-[![\[Telegram\] aiogram live](https://img.shields.io/badge/telegram-aiogram-blue.svg?style=flat-square)](https://t.me/aiogram_live)
+[![Financial Contributors on Open Collective](https://opencollective.com/aiogram/all/badge.svg?label=financial+contributors)](https://opencollective.com/aiogram) [![\[Telegram\] aiogram live](https://img.shields.io/badge/telegram-aiogram-blue.svg?style=flat-square)](https://t.me/aiogram_live)
```
Comment (JrooTJunior) [inline README.md]: This badge should have the same style with other badges.
Can you add `style=flat-square` to badge src?

**Comment by JrooTJunior** [comment]
And changes in `README.md` should be duplicated to `README.rst` and `docs/source/index.rst`

File: README.md (hunk 1 above)
Comment (monkeywithacupcake) [reply README.md]: I made this change

**Pull request merged**